bin/check-ports.rb

```diff
@@ -36,10 +36,10 @@
 # Check Banner
 #
 class CheckPort < Sensu::Plugin::Check::CLI
-  option :host,
+  option :hosts,
```

Comment:
Thanks for trying to remain backwards compatible here and not renaming the external options (api). I think maybe we should add a new option with the new behavior and in the old one we still split it so it becomes an array and log a deprecation warning to the user to switch over. We would keep this deprecation message around for a bit and then release a major change removing the old option. What are your thoughts?

Reply:
So what you suggest is a flag to say whether its one host or multiple hosts?  Not 100% sure what you mean

Reply:
Yes the idea is we release with `--host` only supporting one host and `--hosts` supporting both then after we release that  I will add a deprecation warning in the nex minor release and remove the `--host` option in the next release.

Reply:
Some pin on the latest major and as such they will pull in new minor versions so we want them to pull in the warning so they can see it and prepare for it in the next major release.

Reply:
I really don't see much of a need if the external option remains the same, a comma separated list should suffice, adding a second option just increases complexity and this implementation is backwards compatible